.github/workflows/build-and-deploy.yml

```
@@ -2,7 +2,16 @@ name: build and push
 on: 
   push:
     branches:
-      - '*'
+      - main
```

```suggestion
      - '*'
```
Vi ønsker at alle brancher skal trigge denne pipelinen.